### ISSUE TITLE
parentheses/brackets flagging up in regex strings

### ISSUE THREAD

**Issue by Teifion** [rrrene/credo#290]
I have a multiline regex and it uses parentheses in several parts and it brings up an error.

Reproduction is easy:

`mix new credotest`
add credo to the mix.exs deps
set lib/credotest.ex to the following

```
defmodule Credotest do
  def example_function(s) do
    (1 + 1) * (2 + 2)
  end
  
  def regex_function(s) do
    ~r/(?<cov>[0-9\.]+)%\s+
      (?<file>web\/[_a-zA-Z\/\.\ ]+\/[_a-zA-Z\/\.\ ]+\.ex)\s+
      (?<lines>[0-9]+)\s+
      (?<relevant>[0-9]+)\s+
      (?<missed>[0-9]+)/x
    |> Regex.named_captures(
      String.strip(s)
    )
  end
end
```

Result of `mix credo`
```
  Code Readability                                                              
┃ 
┃ [R] → Modules should have a @moduledoc tag.
┃       lib/credotest.ex:1:11 (Credotest)

  Consistency                                                                   
┃ 
┃ [C] ↗ There is no whitespace around parentheses/brackets most of the time, but 
┃       here there is.
┃       lib/credotest.ex:8 (Credotest.regex_function)

Please report incorrect results: https://github.com/rrrene/credo/issues

Analysis took 0.05 seconds (0.00s to load, 0.05s running checks)
3 mods/funs, found 1 consistency issue, 1 code readability issue.
```

I tried removing the indentation of the regex lines but it doesn't change anything.

**Comment by rrrene** [comment]
Ouch. Thx reporting! :+1:

**Comment by devonestes** [comment]
@Teifion Just so I'm sure, can you tell me what versions of Elixir and Credo that you're using?

**Comment by Teifion** [comment]
```
Erlang/OTP 19 [erts-8.0] [source] [64-bit] [smp:8:8] [async-threads:10] [kernel-poll:false]

Interactive Elixir (1.3.3) - press Ctrl+C to exit (type h() ENTER for help)
```

and 

```
{:credo, "~> 0.5"}
```

**Comment by rrrene** [comment]
@Teifion @devonestes This was fixed by 19cbf03 and will be part of the next release.
